src/assets/css/wp-conference-schedule-pro.css

```diff
@@ -47,6 +47,13 @@
     margin: 20px;
   }
 }
+
+@media (min-width: 782px) {
+    .talk-tags-wrapper .wp-block-columns {
+        flex-wrap: wrap !important;
```

Comment:
Let's avoid using !important unless it's absolutely necessary. This should be manageable by ordering the rules appropriately.